papermill/tests/notebooks/test_logging.ipynb

```diff
@@ -76,22 +76,20 @@
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": [
-    ""
-   ]
+   "source": []
   }
  ],
  "metadata": {
   "hide_input": false,
   "kernelspec": {
-   "display_name": "Python 2",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "python2"
+   "name": "python3"
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 2.0
+    "version": 2
```

Comment:
I'm not sure if the IPython CodeMirror modes have versions that are analogous to the Python version. I assume your notebook UI was responsible for this rewrite and there is not much you can do here but seeing it sparked my curiosity about these modes.

Reply:
Yeah, I just updated notebooks to python 3 kernel by loading them in my virtual env with classic. I'll look into how that one in particular behaves -- it's kind of good to have a variety of different sourced notebooks in the tests to help discover how these interact.